frontend/store/network.ts

```diff
@@ -114,20 +122,36 @@ export const actions = {
         await dispatch('surplus/setup', undefined, { root: true });
         await dispatch('debt/setup', undefined, { root: true });
     },
-    async setupNetworks({ commit }: ActionContext<State, State>, isDev?: boolean) {
+    async setupNetworks({ getters, commit, dispatch }: ActionContext<State, State>, oldRpcUrl?: string) {
```

Comment:
Why do you need to pass `oldRpcUrl` as an argument? Wouldn't it be stored in the store at this point?